spec/lib/has_configuration/configuration_spec.rb

```diff
@@ -11,35 +11,35 @@
     ).to receive(:raw_file).and_return(File.read(fixture))
   end
 
-  describe ".new" do
+  describe '.new' do
     let(:fixture) { 'spec/fixtures/class.yml' }
 
-    context "when no filename provided" do
-      let(:file) { "/RAILS_ROOT/config/class.yml" }
+    context 'when no filename provided' do
+      let(:file) { '/RAILS_ROOT/config/class.yml' }
 
-      it "loads default file" do
+      it 'loads default file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.